content/22.vaccines.md

```diff
@@ -27,102 +27,38 @@ As the SARS-CoV-2 virus has evolved, they also provide insight into how these te
 The development of vaccines is widely considered one of the most important medical advances in human history.
 The past 150 years have seen a rapid diversification in the approaches of vaccine development available [@doi:10.1073/pnas.1400472111].
 Since the turn of the millennium, particular interest has emerged in the potential to develop vaccines as a rapid response to emerging viral threats.
-Recently, such threats have included severe acute respiratory syndrome (SARS), "swine flu" (H1N1), Middle East respiratory syndrome (MERS), and Ebola virus disease (EVD), all of which have underscored the importance of a rapid global response to a new infectious virus.
+Such threats have included severe acute respiratory syndrome (SARS), "swine flu" (H1N1), Middle East respiratory syndrome (MERS), and Ebola virus disease (EVD), all of which have underscored the importance of a rapid global response to a new infectious virus.
 However, vaccine development has historically been slow, and because vaccines fail to provide immediate prophylactic protection or to treat ongoing infections, their application to most of these epidemics has been limited [@doi:10.3390/v10120680].
```

Comment:
```suggestion
However, vaccine development has historically been slow, and because vaccines fail to provide immediate prophylactic protection or treatment of ongoing infections, their application to most of these epidemics has been limited [@doi:10.3390/v10120680].
```